### PR TITLE
Enhanced the ru_RU translation

### PR DESCRIPTION
**Pull request by Agvantibo**
It looked like Google translate and was broken in several places, now it doesn't!

### CODE REVIEW

**Comment by Agvantibo** [comment]
That putin_vladimir guy probably didn't knew what he was doing.
:(

**Comment by Ximi1970** [comment]
Thank you for the translations update. Will leave it parked for now untill I finish the javascript refactor.

File: app/SysTray-X/SysTray-X-app/languages/SysTray-X.ru.ts [hunk 1]
```diff
@@ -42,20 +42,20 @@
     <message>
         <location filename="../aboutdialog.ui" line="82"/>
         <source>Branch:</source>
-        <translation>Ветка:</translation>
+        <translation>Ответвление:</translation>
```
Comment (andreymal) [inline app/SysTray-X/SysTray-X-app/languages/SysTray-X.ru.ts:45]: Ветки в git все всегда называли именно ветками, а не ответвлениями

**Comment by Ximi1970** [comment]
I updated the translations in an other branch. Will be merged to develop soon.

**Pull request closed without merging**